documentation/faqs.md

```diff
@@ -2,17 +2,22 @@
 title: FAQs
 ---
 
-## Is vSphere Cloud Provider ready for production?
-vSphere Cloud Provider is still in beta.
-
 ## What is the biggest Kubernetes cluster it has been tested for?
-It has been tested on eight node cluster so far.
+It has been tested on 500 node cluster so far. Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/largescaledeployment.html) for details about recommended configuration.
```

Comment:
Will it make more clear with "500 node kubernetes cluster"?
In our environment, we have ESXi cluster, vSAN cluster, cabinets cluster.. too many clusters!

Reply:
fixed.